communication.rst

```diff
@@ -92,28 +92,127 @@ RSS feed readers.
-----------------------------------------
+Discourse
+-----------
+**(discuss.python.org web forum)**
 
 We have our own `Discourse`_ forum for both developers and users. This forum
 complements the `python-dev`_, `python-ideas`_, `python-help`_, and
 `python-list`_ mailing lists.
 
-Most core development discussions take place in the open forum categories for
-`PEPs`_ and `Core Development`_. (These are the Discourse equivalents to the
-python-dev mailing list)
-
-The `Committers`_ category restricts posting to core developers only, and is
-used more for announcements and notifications, rather than for discussions. It
-is also the venue for core developer promotion votes. (This category is the
-equivalent of the python-committers mailing list)
-
-There is a final, rarely used, core development category on Discourse that is
-only visible to core developers. This can be used to share administrative
-information with all core developers in a non-public forum (e.g. logistics for
-in person core development sprints), as well as for individual core developers
-to share info that they'd like other core devs to be aware of, but would prefer
-not to have permanently archived on the internet.
+This forum has many different categories and most core development discussions
+take place in the open forum categories for `PEPs`_ and `Core Development`_.
+(These are the Discourse equivalents to the python-dev mailing list)
+Most categories are open for all users to read and post with the exception of
+Committers and Core Development categories.
+
+
+**Committers category**
+
+The `Committers`_ category is an "argument clinic" for Python committers with
+the commit bit on `CPython <https://github.com/python/cpython>`_ and related
+projects.
+It is open for all users to read but posting is restricted to core developers.
+It is often used for announcements and notifications. It is also a common venue
+for the core developer promotion votes (This category is equivalent to the
+python-committers mailing list).
+
+**Core Development category**
+
+The `Core Development`_ category on Discourse is a private category visible to
+core developers only. This category is used to share administrative information,
+as well as individual information inputs among all core developers in an
+enclosed environment where information will not be permanently archived on the
+internet (e.g. logistics for in-person core development sprints).
+
+Getting Started
+''''''''''''''''
+To start a topic or participate in any discussions in the forum, sign up and
+create an account using an email address or Github account. You can do so by
+clicking the "Sign Up" button on the top right hand corner of the `Discourse`_
+main page.
+
+
+Quick Start
+''''''''''''
+The Python Discourse `Quick Start <https://discuss.python.org/t/python-discourse-quick-start/116>`_
+compiled by `Carol Willing <https://discuss.python.org/u/willingc/>`_ gives you
+a quick overview on how to kick off Python Discourse.
```

Comment:
I guess these two can be merged? Possibly even incorporate it in the subsequent section, "Tutorials for New Users".